File: test/fixtures/output/go/native/cookies.go
package main

import (
	"fmt"
	"net/http"
)

func main() {
	client := &http.Client{}
	url := "http://mockbin.com/har"
	req, _ := http.NewRequest("POST", url, nil)
	req.Header.Add("Cookie", "foo=bar; bar=baz")
	res, _ := client.Do(req)
	fmt.Printf("%+v", res)
}

File: test/fixtures/output/ruby/native/cookies.rb
require 'uri'
require 'net/http'

url = URI("http://mockbin.com/har")

conn = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Post.new(url)
request["cookie"] = 'foo=bar; bar=baz'

response = conn.request(request)
puts response.read_body
